packages/calcite-components/src/components/stepper/resources.ts

```diff
@@ -0,0 +1,14 @@
+export const ITEM_MIN_WIDTH = {
+  s: 120,
```

Comment:
Should these sizes be defined by any tokens?

Reply:
yeah they will be defined by tokens. Added an issue https://github.com/Esri/calcite-design-system/issues/8174